akka-stream/src/main/scala/akka/stream/impl/fusing/GraphInterpreter.scala

```diff
@@ -98,12 +102,12 @@ private[stream] object GraphInterpreter {
 
       for (i ← 0 until connectionCount) {
```

Comment:
agreed. This should be not noticable though in the throughput test, since this is initialization code. We probably will need a "latency" benchmark as well, which measures the time from init to the first element downstream.